pkg/volume/glusterfs/glusterfs.go

```diff
@@ -1009,5 +1016,18 @@ func parseClassParameters(params map[string]string, kubeClient clientset.Interfa
 		return nil, fmt.Errorf("StorageClass for provisioner %q must have gidMax value >= gidMin", glusterfsPluginName)
 	}
 
+	if len(parseVolumeOptions) != 0 {
```

Comment:
Thinking ahead to when we (likely) move in-tree plugins out-of-tree, why should kubernetes be parsing these options? Would it be better to pass this []string as is to the provider?